utils/config/nodes.hpp

```diff
@@ -153,36 +124,19 @@ class typed_leaf_node : public leaf_node {
     /// The type of the value held by this node.
     typedef ValueType value_type;
 
+    /// Constructor
     typed_leaf_node(void);
 
+    /// Is the object referred to by this class set?
```

Comment:
Very strange wording, and I'm surprised this works given that this is missing the `\return` stanza that you added everywhere else. Why isn't Doxygen complaining?

Reply:
Yeah, I should have just copied the summary line from the .cpp file.